core/controllers/creator_dashboard_test.py

```diff
@@ -73,10 +73,8 @@ class HomePageTests(test_utils.GenericTestBase):
     def test_logged_out_homepage(self):
         """Test the logged-out version of the home page."""
 
-        response = self.get_html_response('/', expected_status_int=302)
-        response_path = response.headers['location']
-        self.assertEqual(response_path[-1], '/')
-        self.assertEqual(len(response_path.split('/')), 4)
+        response = self.get_html_response('/')
+        self.assertEqual(response.status_int, 200)
```

Comment:
Isn't this covered in another test (test_redirect_in_logged_out_states)? If so, maybe we don't need this one.

Reply:
Yes, done.